InnerEye/Common/fixed_paths.py

```diff
@@ -51,6 +51,11 @@ def repository_root_directory(path: Optional[PathOrString] = None) -> Path:
 # This file is expected to live in the root folder of the repository or project.
 PRIVATE_SETTINGS_FILE = "InnerEyePrivateSettings.yml"
 
+# Secret names for secrets stored as env variables or in PROJECT_SECRETS_FILE
```

Comment:
docu not correct. This is the pass phrase for the Service Principal

Reply:
while you're on that: maybe name the variable (not its contents!) to SERVICE_PRINCIPAL_KEY. We speak of service principal in the documentation, applicationID is an Azure-internal term